### DISCUSSION TITLE
Multi file upload resolver

### DISCUSSION THREAD

**Discussion by snaplesec** [flash-oss/graphql-upload-minimal#8] [Q&A] [answered]
In setting this up with apollo-express server.

How do I setup a multifile upload resolver?

I have the middleware setup for express, but not sure how the resolver is configured for a multifile upload.

**Comment by koresar** [comment, marked as answer]
If you mean "multiple files upload" then there is an example with Express.js in README.

**Reply by snaplesec** [reply]
Thank you, I will look at that, and see what I can get working.
